playbooks/files/pebbles-heat-stack.yml

```diff
@@ -15,7 +15,7 @@ parameters:
     description: >
       Which ports to allow connections to on thejump_host host.
     type: comma_delimited_list
-    default: '22'
```

Comment:
I think we should leave 22 here, because isn't that almost always the port that we are opening anyway?

Reply:
But as for ports, I think port number in configuration should be enough and remove defaults altogether. (Also this way we will have exact configuration details in one place and not mix with defaults ?)

Reply:
Can be done that way. However, ports are essentially static and never change from one deployment to another, so that is why I think the defaults could be left there.